cves/CVE-2013-0870.yml

```diff
@@ -131,7 +139,12 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
```

Comment:
I noticed that there are four vccs auto-identified so now I'm wondering why / how that happened. Maybe you can mention the differences between those vccs in the lifetime if there are any.